src/components/Card/StyledCard.js

```diff
@@ -92,16 +92,16 @@ const CardHeader = ({response, type, expanded}) => {
     <div className='w-100'>
       {headerComponent}
     </div>
-  )  
+  )
 }
 CardHeader.propTypes = {
   response: PropTypes.object,
   type: PropTypes.string.isRequired
 }
 
-const CardBody = ({title, instructor}) => {
+const CardBody = ({title, instructor, type}) => {
```

Comment:
If `type` has a finite set of options, would be better to make it `type: PropTypes.oneOf(types)` where types is an array of the acceptable types